src/concolic/concolic_wasm_ffi.ml

```diff
@@ -93,6 +95,12 @@ module M :
     let c = Value.I32.to_bool i in
     Concolic_choice.assertion c
 
+  let symbol_range (lo : Value.int32) (hi : Value.int32) : Value.int32 Concolic_choice.t =
+    let* x = symbol_i32 () in
+    let* () = assume (Value.Bool.int32 (Value.I32.le lo x)) in
+    let* () = assume (Value.Bool.int32 (Value.I32.gt hi x)) in
+    Concolic_choice.return x
+
```

Comment:
I can't comment there, but you also have to add the newly added function to the list below: https://github.com/OCamlPro/owi/pull/530/files#diff-655f5bc7ec306dcbfee9902f293c2ebee139cdcfa8e1f901a0c8126b96ca35cfR150-R181